package.json

```diff
@@ -110,6 +110,11 @@
         "title": "Manually Provide Authentication Response",
         "category": "GitHub Pull Requests"
       },
+      {
+        "command": "auth.logout",
+        "title": "Log out of GitHub",
```

Comment:
I think this should be "Sign out" since its opposite command is "Sign in"

Reply:
Done.